content/en/blog/_posts/2023-01-06-unhealthy-pod-eviction-policy-for-pdb.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Eviction policy for unhealthy pods guarded by PodDisruptionBudgets"
+date: 2023-01-06
+slug: "unhealthy-pod-eviction-policy-for-pdbs"
+---
+
+**Authors:** Filip Křepinský (Red Hat), Morten Torkildsen (Google), Ravi Gudimetla (Apple)
+
+
+Ensuring the disruptions to your applications do not affect its availability isn't a simple
+task. With the introduction of _unhealthy pod eviction policy_ for [PodDisruptionBudgets](/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) (PDBs),
+this is going to change.
```

Comment:
”going to change” is what puzzles me. 1.26 is already out.

Reply:
https://github.com/kubernetes/website/pull/37627#discussion_r1060182380

Reply:
```suggestion
has changed.
```

That's a simple rewrite; you can use other words, so long as we avoid the incorrect use of future tense.

Reply:
using the other option now: https://github.com/kubernetes/website/pull/37627#discussion_r1060794688